regression/goto-instrument/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ else()
     set(is_windows false)
 endif()
 
-if("{CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
```

Comment:
This looks like a bugfix which should be squashed into the commit that introduced it.